scripts/get-envoy.bash

```diff
@@ -42,6 +42,7 @@ if [ -f "$_dir/envoy" ]; then
 fi
 
 mkdir -p "$_dir"
-curl -L --compressed -o "$_dir/envoy" "https://github.com/pomerium/envoy-binaries/releases/download/v${_envoy_version}/envoy-${_target}"
+curl -L --compressed -o "$_dir/envoy" \
```

Comment:
can we please do arch-platform the way console does? 
when developing on mac I need switch between local and docker environments and it helps to avoid errors like wrong binary being embedded

Reply:
I don't know what that means. If you want a linux binary set `GOOS=linux`.

Reply:
I mean save target as `"$_dir/envoy-${_target}"`

Reply:
Yes, I think the binaries need to retain their platform name upon download so they don't overlap during build.  Previously we were able to put them in the build directory so it wasn't an issue, but embed requires the file to be inline with the code.  Since goreleaser builds the platforms concurrently, it can wind up picking up the wrong binary.  

Concretely, I tried a release snapshot and I'm getting an execv error for envoy when I build on OSX and then run the linux binary:

`
{"level":"error","error":"error starting envoy: fork/exec /tmp/.pomerium-envoy3444331264/envoy: exec format error","service":"envoy","time":"2021-06-10T12:00:51Z","message":"envoy: failed to run envoy process"}
`

I think the name overlap is the cause.